examples/municipality/model/GROW_withATES_Prod_install.esdl

```diff
@@ -38,12 +38,9 @@
         <port xsi:type="esdl:OutPort" id="bd8a6c48-a664-4f32-bf2e-68046a3bfeb4" connectedTo="cc9f9cec-d565-422a-b645-29c24f85678d" carrier="39ec9ed1-30d6-40db-83a4-1bac7b0b3c75" name="Out"/>
         <port xsi:type="esdl:InPort" connectedTo="f7d48b4e-2348-4b24-9af9-98598f270be2" id="3aacb2a9-615b-4047-8b01-04f5f1f32379" carrier="39ec9ed1-30d6-40db-83a4-1bac7b0b3c75_ret" name="In"/>
         <costInformation xsi:type="esdl:CostInformation" id="b0498a6e-b4d6-4ab5-96a7-c4e3918a3edb">
-          <installationCosts xsi:type="esdl:SingleValue" value="300000.0" id="9c6bb73b-8430-49e8-85a1-b3d85d6669cb">
+          <installationCosts xsi:type="esdl:SingleValue" value="36300000.0" id="9c6bb73b-8430-49e8-85a1-b3d85d6669cb">
```

Comment:
Why was this value increased dramatically?

Reply:
The change is good, but should not have been part of this PR. 

We do this as the investmentcosts were used previously, but for geothermal we want the cost to scale with aggregation count, for that we use the installationcost

so basically investmentcost were transferred to installation cost